extide/gradle/apichanges.xml

```diff
@@ -83,6 +83,34 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="gradle-java-independence">
+            <api name="general"/>
+            <summary>Remove Java Platform dependency.</summary>
+            <version major="2" minor="3"/>
+            <date year="2020" month="7" day="15"/>
+            <author login="lkishalmi"/>
+            <compatibility binary="compatible" source="compatible"/>
+            <description>
+                <p>
+                    Due to recent movement and the plan to support Gradle native
+                    projects as well in the future the dependency on
+                    Java Platform module has been removed.
+                </p>
+                <p>
+                    The removal affects <code>org.netbeans.modules.gradle.api.execute.RunUtils</code>
```

Comment:
Btw. it is possible to use `<a href="@TOP@/org/netbeans/modules/gradle/api/execute/RunUtills.html">RunUtils</a>` to connect to the Javadoc generated classes.

Reply:
Thanks! I'm going to try to keep that in mind.